packages/web-js-sdk/src/enhancers/withPersistTokens/helpers.ts

```diff
@@ -18,13 +18,13 @@ import {
 function setJwtTokenCookie(
   name: string,
   value: string,
-  { cookiePath, cookieDomain, cookieExpiration }: Partial<JWTResponse>
+  { cookiePath, cookieExpiration }: Partial<JWTResponse>,
 ) {
   if (value) {
     const expires = new Date(cookieExpiration * 1000); // we are getting response from the server in seconds instead of ms
+    // Since its a JS cookie, we don't set the domain because we want the cookie to be on the same domain as the application
     Cookies.set(name, value, {
       path: cookiePath,
-      domain: cookieDomain,
```

Comment:
need to check if cookie domain is equal or "parent" domain of application domain
e.g. cookie domain - b.com, app domain a.b.com - should work

Reply:
done, @nirgur lmk what you think